### PR TITLE
DC Motor Support

### PR DESCRIPTION
**Pull request by leandrumartin**
Fixes Issue #255. Added helper and controller classes for a DC motor. The classes can control a DC motor's speed. Planned are direction controls (forward/reverse) and tests for the helper class.

The structure of the code added follows the basic set structure for adding components to the repository: `application.yml` specifies what pins to use for the motor, the helper class is added under `pi4micronaut-utils`, and the controller class is added under `components/controllers`.

I chose to rename the `MotorController.java` as specified in the issue description to `DCMotorController.java` to more easily differentiate it from the similarly named servo motor component.

### CODE REVIEW

**Comment by leandrumartin** [comment]
I have named the classes back to `MotorController`. I've now also added the API and method for changing the motor speed. I have added the API for changing the direction and stubbed out the method implementing that functionality.

File: components/src/main/resources/application.yml [hunk 1]
```diff
@@ -35,6 +35,13 @@ pi4j:
       provider: pigpio-pwm
       initial: 0
       shutdown: 0
+    dc-motor:
```
Comment (Ed0827) [inline components/src/main/resources/application.yml:38]: The structure in application.yml looks great! I’m just curious—could the address be a different number? Is it possible to choose the address randomly?

File: pi4micronaut-utils/src/main/java/com/opensourcewithslu/outputdevices/MotorHelper.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        // TODO: Implement logic to set the direction of the DC motor.
+
+        isClockwise = clockwise;
+    }
+
+    /**
+     * Switches the direction of the DC motor.
+     */
+    public void switchDirection() {
+        setClockwise(!isClockwise);
```
Comment (Ed0827) [inline pi4micronaut-utils/src/main/java/com/opensourcewithslu/outputdevices/MotorHelper.java:90]: Well written description of each method and well implemented them as well.

**Review by Ed0827 — COMMENTED** [review]
Overall, Leandru has successfully implemented the MotorController, MotorHelper, and application.yml. The files have been placed correctly, with MotorController in the inputdevice folder and MotorHelper in the outputdevice folder.

**Comment by kate-holdener** [comment]
@leandrumartin Do you also need to add a motorController.adoc file to `pi4micronaut-utils/src/docs/asciidoc/components/inputComponents/` directory?

**Comment by leandrumartin** [comment]
The direction setting is now complete and the `MotorHelper` class now has tests implemented for all methods.

**Pull request merged**